README.md

```diff
@@ -48,39 +61,92 @@ npm install @asgardeo/auth-nodejs-sdk
 
 ## Getting Started
 
+###### **This getting started guide is an example of [Express JS](https://expressjs.com/) implementation. Other Node Platforms may differ from this implementation**
+
 ```javascript
+// Import the packages needed to create an Express app
+const express = require("express");
+const cookieParser = require("cookie-parser");
+const { v4: uuidv4 } = require("uuid");
+
+// Import the Asgardeo Node SDK
 // The SDK provides a client that can be used to carry out the authentication.
 const { AsgardeoAuth } = require('@asgardeo/auth-nodejs-sdk');
 
 // Create a config object containing the necessary configurations.
 const config = {
-    signInRedirectURL: "http://localhost:3000/sign-in",
-    signOutRedirectURL: "http://localhost:3000/login",
-    clientID: "client ID",
-    serverOrigin: "https://api.asgardeo.io/t/<org_name>"
+    clientID: "<your_client_id>",
+    clientSecret: "<your_client_secret>",
+    serverOrigin: "https://api.asgardeo.io/t/<org_name>",
+    signInRedirectURL: "http://localhost:3000/auth/login",
+    signOutRedirectURL: "http://localhost:3000",
+    scope: ["openid", "profile"]
 };
 
+// Create an express app  and setup
+const PORT = 3000;
+const app = express();
+app.use(cookieParser());
+app.set("view engine", "ejs");
+app.use("/", express.static("static"));
+
 // Instantiate the AsgardeoAuthClient and pass the config object as an argument into the constructor.
 const authClient = new AsgardeoAuth(config);
 
-//If you are using ExpressJS, you may try something like this.
-app.get("/login", (req, res) => {
+// Define the data template needed for the view engine
+const dataTemplate = {
+    isConfigPresent: Boolean(config && config.clientID && config.clientSecret),
+    isAuthenticated: false,
+    idToken: null,
+    error: false,
+    authenticateResponse: null
+}
```

Comment:
Yes but since the login route has a redirection to the `/` route, should we include that route as well? I added this because I wanted to clarify on the same and extend the getting started guide with `/` endpoint.